Makefile

```diff
@@ -1,3 +1,4 @@
 setup_dev:
 	pip install -r requirements.txt
 	pre-commit install
+	pip install -e .
```

Comment:
Using this may cause inconsistency.  Do you need to edit the code for installed packages?